install.sh

```diff
@@ -314,18 +315,18 @@ ln -s "../lib/crew/crew" "${CREW_PREFIX}/bin/"
 # prepare sparse checkout .rb packages directory and do it
 cd "${CREW_LIB_PATH}"
 rm -rf .git
-git init
-git remote add -f origin "https://github.com/${OWNER}/${REPO}.git"
-git config core.sparsecheckout true
+$CREW_PREFIX/bin/git init
+$CREW_PREFIX/bin/git remote add -f origin "https://github.com/${OWNER}/${REPO}.git"
+$CREW_PREFIX/bin/git config core.sparsecheckout true
```

Comment:
There is only one git so no need to use the full path to the binary.

Reply:
I was wondering why this broke the script, and it turns out that the extract function doesn't extract to the new CREW_PREFIX. See the last commit.